core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/ProxyV4Controller.java

```diff
@@ -58,6 +58,13 @@ public ProxyV4Response delete(Long workspaceId, String name) {
         return converterUtil.convert(deleted, ProxyV4Response.class);
     }
 
+    @Override
+    public ProxyV4Responses deleteMultiple(Long workspaceId, Set<String> names) {
+        Set<ProxyConfig> deleted = proxyConfigService.deleteMultipleByNameFromWorkspace(names, workspaceId);
+        notify(ResourceEvent.PROXY_CONFIG_DELETED);
```

Comment:
I think we will send false information in some case.
Do we delete all or nothing? Can we delete a particular set of names?
I mean, we get 3 names, name1, name2, name3. We delete successfully name1, name2, but name3 failed. Do we send success in this case?

Reply:
In this case we would send a failure -- this is a problem that exists outside of this commit, however. I'd need to look into how CB uses transactions, but I would think annotating `delete(Set<T> resources)` in `AbstractWorkspaceAwareResource` service as `@Transactional` would suffice here.

Reply:
So, any reason we can't address this in a follow on JIRA? UI is waiting on this functionality, and this would require a bit more testing to make sure things are behaving as expected.